scripts/observers/config.py

```diff
@@ -0,0 +1,113 @@
+import os
+import toml
+import sys
+import shutil
+from pathlib import Path
+from git import Repo
+from dotenv import load_dotenv
+
+load_dotenv()
+
+observer_dir = str(os.getenv('OBSERVER_DIR'))
+METACHAIN = 4294967295
+
+
+def update_toml_indexer(path, shard_id):
+    # prefs.toml
+    data1 = toml.load(path+"/prefs.toml")
```

Comment:
Bit of renaming: `data1` vs. `prefs_data` or something similar.

Reply:
renamed